integration-tests/src/test/java/oracle/kubernetes/operator/ItModelInImageOverride.java

```diff
@@ -0,0 +1,246 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator;
+
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Level;
+
+import oracle.kubernetes.operator.utils.Domain;
+import oracle.kubernetes.operator.utils.DomainCrd;
+import oracle.kubernetes.operator.utils.ExecResult;
+import oracle.kubernetes.operator.utils.LoggerHelper;
+import oracle.kubernetes.operator.utils.Operator;
+import oracle.kubernetes.operator.utils.Operator.RestCertType;
+import oracle.kubernetes.operator.utils.TestUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Simple JUnit test file used for testing Model in Image.
+ *
+ * <p>This test is used for creating domain using model in image.
+ */
+
+public class ItModelInImageOverride extends MiiBaseTest {
+  private static Operator operator;
+  private static Domain domain;
+  private static String domainNS;
+  private static String testClassName;
+  private static StringBuffer namespaceList;
+  private static final String configMapSuffix = "-mii-config-map";
+
+  /**
+   * This method gets called only once before any of the test methods are executed. It does the
+   * initialization of the integration test properties defined in OperatorIT.properties and setting
+   * the resultRoot, pvRoot and projectRoot attributes.
+   *
+   * @throws Exception exception
```

Comment:
explain why an exception might be thrown, we already know it is an exception

Reply:
change to:

  * @throws Exception exception if initializing the application properties 
   *          and creates directories for results fails.